src/main.ts

```diff
@@ -25,8 +25,9 @@ import { processSources } from './processor';
     warnings.forEach(({ location, messages }) => {
         console.warn(location);
         messages.forEach(message => {
-            console.warn('  ' + message);
+            console.warn(`  ${message}`);
```

Comment:
why not removing the empty `console.warn()` and having:
console.warn(` ${message}\n`);